packages/iproute/iproute.spec

```diff
@@ -76,9 +68,7 @@ done
 %{_cross_sbindir}/vdpa
 %dir %{_cross_libdir}/tc
 %{_cross_libdir}/tc/*
-%dir %{_cross_factorydir}%{_cross_sysconfdir}/iproute2
-%{_cross_factorydir}%{_cross_sysconfdir}/iproute2/*
-%{_cross_tmpfilesdir}/iproute2.conf
+%{_cross_datadir}/iproute2/*
```

Comment:
nit: I like to make sure a package has explicit ownership of its own directories:
```
%dir %{_cross_datadir}/iproute2
%{_cross_datadir}/iproute2/*
```